packages/components/src/components/validate/validate.component.ts

```diff
@@ -0,0 +1,268 @@
+    const input = this.getInput();
+
+    // If there is no input, skip before doing any harm
+    if (!input) {
+      return [];
+    }
+
+    // Make sure to always use an array of events
+    // This is needed because on may be a special value like "live"
+    const on = Array.isArray(this.on) ? this.on : [this.on];
+
+    // Filter makes sure to remove empty values, e.g.
+    // <syn-validate on=""></syn-validate>
+    const [...events] = on.filter(Boolean);
+
+    const isSynergyElement = input instanceof SynergyElement;
+
+    // Make sure to always have an invalid event
+    if (!events.includes('invalid')) {
+      events.push('invalid');
+    }
+
+    // Special handling for the live keyword:
+    // live always means on input and blur
+    if (events.includes('live')) {
+      events.push('input');
+      events.push('blur');
+    }
+
+    // Make sure to remove duplicated events and the live property
+    const finalEvents = Array.from(new Set(events.filter(e => e !== 'live')));
+
+    // If the input is a synergy element, use syn- prefixed events.
+    // Only adjust items that do not already have syn- prefixes to
+    // prevent event names like syn-syn-input from being added.
+    if (isSynergyElement) {
+      return finalEvents.map(e => {
+        if (e.startsWith('syn-')) return e;
+        return `syn-${e}`;
+      });
+    }
+    return finalEvents;
+  }
+
+  /**
+   * Update the events on the input element.
+   */
+  private updateEvents() {
+    this.controller.abort();
+    this.controller = new AbortController();
+    const input = this.getInput();
+
+    if (!input) {
+      return;
+    }
+
+    const events = this.getUsedEventNames();
+    events.forEach(eventName => {
+      input.addEventListener(eventName, this.validate, {
+        capture: isInvalidEvent(eventName),
+        signal: this.controller.signal,
+      });
+    });
+  }
+
+  private setValidationMessage(input: HTMLInputElement) {
+    const { customValidation } = this;
+    const validationMessage = customValidation || input.validationMessage;
+
+    this.validationMessage = validationMessage;
+  }
+
+  /**
+   * Handle the blur event during validation
+   */
+  // eslint-disable-next-line class-methods-use-this
+  private handleFocus(input: HTMLInputElement) {
+    const activeElement = document.activeElement! as HTMLInputElement;
+    const activeElementIsWrapped = activeElement.closest('syn-validate');
+
+    if (!activeElement.validity?.valid && activeElementIsWrapped) {
+      // The active element is invalid do not scroll
+      return;
+    }
+
+    input.scrollIntoView({ block: 'nearest' });
+    input.focus();
+  }
+
+  /**
+   * Triggers a validation run, showing the validation message if needed.
+   */
+  private validate = (e: Event) => {
+    // Make sure to always prevent the invalid event when using inline validation
+    if (isInvalidEvent(e.type) && this.inline) {
+      e.preventDefault();
+      e.stopPropagation();
```

Comment:
Problem is you have to stop it, otherwise you will trap the input in case of inline validation. As the event uses event capturing (e.g. the event flows from "the top down" instead of default "from the bottom up" to inform the parent form, it is still possible to listen to the event.

When throwing an invalid event on native inputs, the input will automatically receive focus, this is why this was done.